### DISCUSSION TITLE
Timed conditional access doesn't seem to work

### DISCUSSION THREAD

**Discussion by habi** [osmandapp/OsmAnd#11189] [Q&A] [answered]
A friend of mine made his first OSM edit with my help.
He's using OsmAnd for routing and wanted to correct the routing around https://www.openstreetmap.org/node/5026993763 (which seems correct in my opinion).
His OsmAnd Live subscription does *not* take into account the conditional access, even after some days and at the correct time (as seen in the screenshot).

Before submitting an issue I wanted to ask here in the discussions if OsmAnd should be routing correctly, e.g. if it does take the tags into account.
The bollards are free to pass from 09:00-17:00 and prohibit cars passing from 17:00-09:00

![signal-2021-03-18-113712](https://user-images.githubusercontent.com/1651235/111615235-c572ef00-87e0-11eb-9c7c-1fb901bace40.jpeg)

**Comment by vshcherb** [comment, marked as answer]
It's clearly an issue cause this should be supported since last year. https://github.com/osmandapp/OsmAnd/issues/11191

**Reply by habi** [reply]
Thanks for the clarification and thanks for adding this as an issue.
Let me know if I can be of any help with debugging, I'm happily using the TestFlight version of OsmAnd on iOS, albeit without a live subscription at the moment.

**Comment by habi** [comment]
The 'bug' has been fixed with proper tagging of the bollard: https://osmcha.org/changesets/101979640/
Thanks for the help/input!
